api/config/gateway/v1/gateway.proto

```diff
@@ -85,6 +85,12 @@ message GatewayOptions {
   Timeouts timeouts = 5;
 
   repeated Middleware middleware = 6;
+
+  Frontend frontend = 7;
+}
+
+message Frontend {
+  string cdn_url = 1;
```

Comment:
it's not really a CDN, it's a passthrough. a CDN would go in front of the server whereas this is behind it.

also in our case we use s3, not a publicly accessible URL, so the config will need to a be bit different.

Reply:
Yup makes sense I can change it up. 

Why will the config be different for us? Our IAM role will have access to these assets?

Reply:
we need to use an s3 client in order to sign the requests, a regular HTTP request won't do

Reply:
Yeah i see what your saying now after reading a bit, let me compensate.